### PR TITLE
Add image management docs

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

Add docs for Advanced image management dashboard

### CODE REVIEW

**Comment by usha-mandya** [comment]
Preview - https://deploy-preview-12502--docsdocker.netlify.app/docker-hub/image-management/

**Review by mark-dr — COMMENTED** [review]
Is it okay to use internal test users for screenshots in the docs?

- scan-e2e isn't just a test repo, it's really used for some of our e2e tests
- `testuserproplan` has a pretty silly (and maybe copyrighted :grimacing:) avatar

It feels a bit like we're showing the users how the sausages are made.

**Comment by usha-mandya** [comment]
Thanks for the review @mark-dr. Didn't want to use the dummy data, so added this screenshot instead. Is there an example repo I can log into to get a screenshot?

**Comment by mark-dr** [comment]
LGTM, thanks for the changes :slightly_smiling_face:

**Comment by usha-mandya** [comment]
Removed the 'Image Management API' section.

File: docker-hub/image-management.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+The Advanced Image Management dashboard provides a snapshot of your existing images and allows you to view, sort and filter images by tags, activity status and date, and contains options to clean up your workspace by deleting images that are no longer required.
+
+The dashboard provides visibility into the status of the images in your repository and allows you to reduce the number of images stored in Docker Hub by reviewing their active status. An image retains its 'active' status if it is pulled or pushed in the last month. If there isn’t any activity on the image in the last month, it is considered 'inactive'.
+
+By displaying the activity status and tags, the dashboard makes it easier for you to review the existing images and take actions to delete images that are not needed anymore. This is particularly helpful when your repository contains a lot of images that you have pushed over time, and you are now trying to streamline storage in Docker Hub.
```
Comment (nebuk89) [inline docker-hub/image-management.md:13]: add in:

You can also see the old versions of images you have pushed. When you push an image to Docker Hub you are pushing a manifest, a list of all of the layers of your image, and the layers themselves. When you are updating an existing tag, only the new layers will be pushed along with the new manifest which references these layers. This new manifest will be given the tag you specify when you push, such as myNamespace/mytag:latest. This does not remove the old manifests or the unique layers referenced by them from Hub.
You can in still use and reference these using the digest of the manifest if you know the SHA.

With the new Advanced Image management Dashboard all of this is now accessible for you to browse, use and clean up.

**Review by nebuk89 — CHANGES_REQUESTED** [review]
Added a section to provide more explanation on what users will be seeing

**Review by FranchommeE — APPROVED** [review]
LGTM

- wording around plan/subscription can be updated later (voice and tone)
- link to Hub API doc will 404 until https://docker.atlassian.net/browse/HUB-4980 PR is merged

**Pull request merged**